### ISSUE TITLE
Feature: add range to virtualized component

### ISSUE THREAD

**Issue by mpodlasin** [oliviertassinari/react-swipeable-views#423]
Currently you can only pass `slideCount` prop, which enforces `[0, slideCount]` range on rendered list. In other words, index is limited from both sides.

What I would like to do is to give user a way to control both ranges, even with setting index limit for one side, but not the other. For example I want to set `[Number.NEGATIVE_INFINITY, 10]`, which would render components with indexes from -infinity to 10, `[-20, 20]` would render indexes from -20 to 20 (41 components) or `[-100, Number.POSITIVE_INFINITY]` which would render components from -100 index to infinity. 

I need it to render dates, which should go arbitrarily far to the past, but should end at current date and not go to the future.

I can make PR if you would like to.

**Comment by rsumner32** [comment]
Yeah that be great I’m trying to recover my passwords from rsumner31 and now rsumner447 crap is ridiculous

**Comment by mpodlasin** [comment]
I'll try to do it today

**Comment by jeanpoelie** [comment]
Are you still planning on making a PR? I can than check it and merge it :)
